apps/go-mod-validator/src/deps.ts

```diff
@@ -0,0 +1,52 @@
+const { execSync } = require("child_process");
+
+function getGoModFiles(): string[] {
+  try {
+    const output = execSync("find \"$(pwd)\" -type f -name 'go.mod'", {
+      encoding: "utf-8",
+    });
```

Comment:
zx looks like something built for writing shell scripts in js style, not sure if I'm right 🤔  

I found only this one [page](https://google.github.io/zx/typescript) about using zx in typescript, where they also mention _Node.js requires additional setup_

They've also mentioned in the documentation it's useful in case we're writing complex shell queries but, we've 2 simple shell queries so I didn't feel we required it.

Reply:
I recommend we use https://github.com/actions/toolkit/tree/main/packages/glob instead

Reply:
glob looks good. Implemented using that